C/ClustalO/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ClustalO"
+version = v"1.2.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/GSLBiotech/clustal-omega.git",
+                  "d21fab82d380638c568c9427ed39cb42dd87d93b"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/clustal-omega
+install_license LICENSE COPYING
+autoreconf -fvi
+mkdir build
+cd build
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+-DARGTABLE_INCLUDE_DIR=$WORKSPACE/destdir/include -DARGTABLE_LIBRARY_DIR=$WORKSPACE/destdir/lib \
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables
```suggestion
-DARGTABLE_INCLUDE_DIR="${includedir}" -DARGTABLE_LIBRARY_DIR="${libdir}" \
```